lib/logster/middleware/viewer.rb

```diff
@@ -141,10 +141,10 @@ def serve_messages(req)
 
         payload = {
           messages: @store.latest(opts),
-          total: @store.count
+          total: @store.count,
+          search: (req['search'] or ''),
+          filter: (filter or '')
```

Comment:
style wise I always prefer || over or, see the full spiel at http://www.virtuouscode.com/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/

Reply:
I have been enlightened.